src/app/shared/view-issue/new-team-response/new-team-response.component.ts

```diff
@@ -100,11 +100,9 @@ export class NewTeamResponseComponent implements OnInit {
       finalize(() => this.isFormPending = false)
     ).subscribe((resultArr: [Issue, IssueComment]) => {
         const [updatedIssue, updatedComment] = resultArr;
-      updatedIssue.teamResponse = this.description.value;
-      updatedIssue.duplicateOf = this.duplicateOf.value === '' ? undefined : this.duplicateOf.value;
       updatedIssue.issueComment = updatedComment;
       this.issueUpdated.emit(updatedIssue);
-      form.resetForm();
+      form.resetForm(this.newTeamResponseForm.value);
```

Comment:
@JunWei96 I haven't yet determined the exact reason for the form entering a "deactivated" state when the response is submitted.

However, I tried to resolve the bug by explicitly resetting the form using the values of the form controls.

With this, the form no longer enters a "deactivated" state when we submit the form with an empty team response.
![optional-desc](https://user-images.githubusercontent.com/35621759/75417791-cd10ed00-596c-11ea-89e6-f1091ca37798.gif)